subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/SwiftProjectInitDescriptor.java

```diff
@@ -107,9 +107,9 @@ protected String getHostTargetMachineDefinition() {
     protected void configureTargetMachineDefinition(ScriptBlockBuilder buildScriptBuilder) {
         if (OperatingSystem.current().isWindows()) {
             buildScriptBuilder.methodInvocation("Swift tool chain does not support Windows. The following targets macOS and Linux:", "targetMachines.add", buildScriptBuilder.propertyExpression("machines.macOS.x86_64"));
-            buildScriptBuilder.methodInvocation(null, "targetMachines.add", buildScriptBuilder.propertyExpression("machines.linux.x86_64"));
+            buildScriptBuilder.methodInvocation("Set the target operating system and architecture for this library", "targetMachines.add", buildScriptBuilder.propertyExpression("machines.linux.x86_64"));
```

Comment:
This was done, to have all build script comments directly on a statement so that they can be replaced by a `// <x>` placeholder when the comment is moved out of the script into the readme when generating samples (see #14291).